backend/utils/airtable/utils.js

```diff
@@ -122,7 +122,12 @@ async function updateAirtableEntry(recordId, proposal, grantCompleted = false) {
     update["Minimum USD Requested"] = proposal.minUsdRequested;
 
   if (proposal.oneLiner) update["One Liner"] = proposal.oneLiner;
-  await base("Proposals").update(recordId, update);
+  try {
+    await base("Proposals").update(recordId, update);
```

Comment:
I don't think a try catch block will work around async function, you should use `.then(error => ... throw error)`

Reply:
You're right but it should catch the error because of `await`
Example code snippet:

```js
const sleep = (ms) => new Promise(resolve => setTimeout(resolve, ms));

async function error(){
    await sleep(3000)
    throw new Error('error')
}

async function catchError(){
    try {
        await error()
    } catch (e) {
        console.log("Error happned:",e)
    }
}

catchError()
```

Reply:
When I tested it, it didn't work. Maybe it's something related with the way Airtable throws the error. But if you tested it and it worked it's fine

Reply:
![image](https://user-images.githubusercontent.com/25263018/163380776-8c376fc3-d5e6-4b69-85fd-b8395ee6debd.png)

Reply:
Ok, I'll approve